primitives/runtime-interface/proc-macro/src/runtime_interface/host_function_interface.rs

```diff
@@ -96,6 +96,7 @@ fn generate_extern_host_function(
 		method.sig.ident,
 	);
 	let return_value = &method.sig.output;
+	let cfg_attrs = method.attrs.iter().filter(|a| a.path().is_ident("cfg"));
```

Comment:
nit: this filtering is repeated multiple time, maybe it is worth putting it to some shared util?